test-library/src/main/java/com/pocs/test_library/mock/Post.kt

```diff
@@ -4,16 +4,19 @@ import com.pocs.domain.model.post.Post
 import com.pocs.domain.model.post.PostCategory
 import com.pocs.domain.model.post.PostDetail
 
-val mockPostDetail1 = PostDetail(1, "", mockPostWriter1, views = 1, "", "2022-08-03", null, PostCategory.NOTICE)
+val mockPostDetail1 =
+    PostDetail(1, "", mockPostWriter1, views = 1, true, "", "2022-08-03", null, PostCategory.NOTICE)
```

Comment:
이렇게 다음줄로 넘어갈때는 아래와 같이 하는게 좋습니다. 그리고 named argument로 전환해도 좋습니다.
```suggestion
    PostDetail(
      1, 
      "", 
      mockPostWriter1,
     ...
```